src/scenes/HomeContainer.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
+import PropTypes from 'prop-types'
+import Header from './components/Header/Header'
+import Body from './components/Body/Body'
+import Footer from './components/Footer/Footer'
+import { getOrganizationDetails } from '../store/actions/organization'
+import { getEventDetails } from '../store/actions/event'
+import HomeWrapper from '../theme/root'
+
+class HomeContainer extends Component {
+  static propTypes = {
+    getOrganizationDetails: PropTypes.func.isRequired,
+    getEventDetails: PropTypes.func.isRequired,
+    event: PropTypes.shape({
+      title: PropTypes.string.isRequired,
+      date: PropTypes.oneOfType([
+        PropTypes.string,
+        PropTypes.instanceOf(Date),
+      ]),
+    }).isRequired,
+  }
+
+  componentDidMount() {
+    this.props.getOrganizationDetails()
+    this.props.getEventDetails(0)
+  }
+
+  render() {
+    const { title, date } = this.props.event
+
+    return (
+      <HomeWrapper>
+        <Header title={title} date={date} />
```

Comment:
The header should also receive the WebMadeira logo by property (which comes from the store at organization details).

Reply:
👍